src/GitHub.VisualStudio/GitHubPackage.cs

```diff
@@ -25,7 +26,7 @@ namespace GitHub.VisualStudio
     [ProvideAutoLoad("11B8E6D7-C08B-4385-B321-321078CDD1F8")]
```

Comment:
To utilize async loading during auto load, you also need AllowsBackgroundLoad parameter set to true here